data/jp/quotes.json

```diff
@@ -660,7 +660,7 @@
 		"Docking(1)": "まだ航行できますが…ご命令ならば。",
 		"Docking(2)": "じっとしてるのは性に合いませんね。",
 		"Join": "水雷戦隊、出撃します！",
-		"Sortie": "水雷戦隊、出撃します！",
+		"Sortie": "期待に応えてみせます。,
```

Comment:
`"` is missing

Reply:
Oops, Sorry.

Reply:
checking still not passed; shall not edit on github web, it changes all lines...

Reply:
I'm sorry for that. I have to learn more how github works...

Reply:
It's okay at all. probably.